exchanges/btse/btse.go

```diff
@@ -187,19 +203,14 @@ func (b *BTSE) GetAccountBalance() (*AccountBalance, error) {
 // CreateOrder creates an order
 func (b *BTSE) CreateOrder(amount, price float64, side, orderType, symbol, timeInForce, tag string) (*string, error) {
 	req := make(map[string]interface{})
-	req["amount"] = strconv.FormatFloat(amount, 'f', -1, 64)
-	req["price"] = strconv.FormatFloat(price, 'f', -1, 64)
+	req["amount"] = amount
+	req["price"] = price
+	if req["side"] != "" {
```

Comment:
This will always be empty as req["side"] hasn't been populated yet. This is also missing a few fields as per here: https://www.btse.com/apiexplorer/spot/#tocs_apiordermodel